fastn-core/src/library2022/processor/sql.rs

```diff
@@ -6,11 +6,20 @@ pub async fn process(
     doc: &ftd::interpreter::TDoc<'_>,
     config: &mut fastn_core::RequestConfig,
     q_kind: &str,
+    preview_session_id: &Option<String>,
```

Comment:
Change the signature to accept a Boolean, which indicates if query should be executed so in future when you do session id check, you do not have to bring that logic to this level.

Reply:
done in https://github.com/fastn-stack/fastn/pull/1945/commits/dc187a525677361928fb7626a0d41f1e46a93608